webapp/channels/src/i18n/en.json

```diff
@@ -5537,7 +5537,6 @@
   "user.settings.advance.deactivateDescShort": "Click 'Edit' to deactivate your account",
   "user.settings.advance.formattingDesc": "If enabled, posts will be formatted to create links, show emoji, style the text, and add line breaks. By default, this setting is enabled.",
   "user.settings.advance.formattingTitle": "Enable Post Formatting",
-  "user.settings.advance.icon": "Advanced Settings Icon",
```

Comment:
This'd also need to be reverted if changing the ID is a mistake in the `UserSettingsModal`

Reply:
done!